src/pages/Recruitment/Recruit.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import RecItem from "./compnents/RecItem";
+
+const Recruit = () => {
+  const [recruit, setRecruit] = useState([]);
+  const [isBtnType, setIsBtnType] = useState({
+    isListType: false,
+    isTileType: false,
+  });
+
+  useEffect(() => {
+    fetch("/data/rec/recruit.json")
+      .then((res) => res.json())
+      .then((data) => setRecruit(data));
+  }, []);
+
+  return (
+    <S.RecruitCont>
+      <S.ListType>
+        <S.TypeListBtn
+          onClick={() => setIsBtnType((prev) => !prev)}
+          ListBgImg={
+            isBtnType === true
+              ? "./images/rec/listType_ico_off.png"
+              : "./images/rec/listType_ico_on.png"
+          }
+        >
+          타입1
+        </S.TypeListBtn>
+        <S.TypeTilBtn
+          onClick={() => setIsBtnType((prev) => !prev)}
+          TileBgImg={
+            isBtnType === true
+              ? "./images/rec/tileType_ico_on.png"
+              : "./images/rec/tileType_ico_off.png"
+          }
+        >
+          타입2
+        </S.TypeTilBtn>
+      </S.ListType>
+      <S.RecListCont>
+        <S.RecList
+          display={isBtnType === true && "flex"}
+          margin={isBtnType === true && "-10px -5px 0 -5px"}
```

Comment:
```suggestion
          display={isBtnType}
          margin={isBtnType}
```
두 타입 값 모두 boolean 값이기 때문에 이와같이 작성하고
스타일은 스타일 컴포넌트에서 적용해 주는게 가독성에 더 좋겠네요!